docs/topics/generic.rst

```diff
@@ -12,7 +12,7 @@ Buyers
 
 Buyers are identified by a UUID, which is a string (max 255 chars) that makes
 sense to the client. It must be unique within solitude, so we'd recommend
-prefixing the UUID, eg: `marketplace:<your-uuid>`
+prefixing the UUID, eg: ``marketplace:<your-uuid>``
```

Comment:
the single ticks make *emphasis* in restructured text, double ticks makes `code`. I think this was just a typo because confusingly single ticks in markdown will make code.